footprints/Espressif.pretty/ESP32-C5-DevKitC-1.kicad_mod

```diff
@@ -0,0 +1,923 @@
+(footprint "ESP32-C5-DevKitC-1"
```

Comment:
Please align the footprint at the origin (0,0) on the first/pin 1 TH pad, like in the other devkits.

![Screenshot 2025-02-04 at 10 20 43](https://github.com/user-attachments/assets/7879aa64-0858-49af-8000-27ef3033363f)